### PR TITLE
Meeting-token migrated to use Live Share SDK - C#

### PR DESCRIPTION
**Pull request by HunaidHanfee-MSFT**

### CODE REVIEW

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Reviewed and provided my comments.

File: samples/meetings-token-app/csharp/ClientApp/src/Context/TeamsContextProvider.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ class TeamsContextProvider extends Component {
         const { PreMeeting, SidePanel } = Constants.Surfaces;
         this.getContext()
             .then((context = {}) => {
+                console.log(context);
```
[the remark this answers is not in the document]
Comment (HunaidHanfee-MSFT) [reply samples/meetings-token-app/csharp/ClientApp/src/Context/TeamsContextProvider.js:22]: Not as required. So, I have removed it.

File: samples/meetings-token-app/csharp/ClientApp/src/Components/TabConfig.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import React from 'react';
+import * as microsoftTeams from "@microsoft/teams-js";
+
+/**
+ * The 'Config' component is used to display your group tabs
```
[the remark this answers is not in the document]
Comment (HunaidHanfee-MSFT) [reply samples/meetings-token-app/csharp/ClientApp/src/Components/TabConfig.js]: Changed to Meeting Tabs

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**